ibis/backends/tests/test_temporal.py

```diff
@@ -631,32 +641,21 @@ def convert_to_offset(offset, displacement_type=displacement_type):
     ],
 )
 @pytest.mark.notimpl(
-    [
-        "datafusion",
-        "flink",
-        "impala",
-        "mysql",
-        "sqlite",
-        "polars",
-        "mssql",
-        "druid",
-        "oracle",
-    ],
+    ["datafusion", "flink", "impala", "sqlite", "polars", "druid"],
     raises=com.OperationNotDefinedError,
 )
 @pytest.mark.notimpl(
-    [
-        "sqlite",
-    ],
+    ["sqlite"],
     raises=(com.UnsupportedOperationError, com.OperationNotDefinedError),
     reason="Handling unsupported op error for DateAdd with weeks",
 )
 @pytest.mark.notimpl(["exasol"], raises=com.OperationNotDefinedError)
 def test_integer_to_interval_date(backend, con, alltypes, df, unit):
     interval = alltypes.int_col.to_interval(unit=unit)
-    array = alltypes.date_string_col.split("/")
-    month, day, year = array[0], array[1], array[2]
-    date_col = ibis.literal("-").join(["20" + year, month, day]).cast("date")
+    month = alltypes.date_string_col[:2]
+    day = alltypes.date_string_col[3:5]
+    year = alltypes.date_string_col[6:8]
+    date_col = ("20" + year + "-" + month + "-" + day).cast("date")
```

Comment:
More backends implement the primitives here than the previous code, which used arrays.